trashapplet/src/trash-empty.c

```diff
@@ -229,8 +229,10 @@ trash_empty_start (GtkWidget *parent)
   GtkBuilder *builder;
   GTask      *task;
   gint i;
+  const gchar *resource_name;
 
-  builder = gtk_builder_new_from_resource (GRESOURCE "trashapplet-empty-progress.ui");
+  resource_name = "/org/gnome/gnome-applets/trash/trashapplet-empty-progress.ui";
```

Comment:
gnome-applets? really?

Reply:
Yeah obviously should be `/org/mate/mate-applets/trash/trashapplet-empty-progress.ui`.

@lukefromdc any reason why you chose to drop the `GRESOURCE` macro?  It's not so pretty, but it allows to unify the prefix used in both places.  I don't care much, but it would likely not have lead to this :)

Reply:
That looks like an accident, trash was the first applet I ported. I will bring that back and see if I can clean up the makefile.am files  a bit too

Reply:
Starting point was gnome-applet's 2016 port to in-process, so "gnome-applets" is a bad cut and paste. Thanks for catching this!

Reply:
Brought it back.